src/components/CalendarGrid.vue

```diff
@@ -124,8 +123,7 @@ export default {
 				navLinks: true,
 				// Localization
 				...getDateFormattingConfig(),
-				locale: getFullCalendarLocale(getLocale(), this.locale),
-				firstDay: getFirstDayOfWeekFromMomentLocale(this.locale),
```

Comment:
is the first day ever set now?!

Reply:
The first day was set in both the getFullCalendarLocale function (dow field) and firstDay before. Just the dow field works, but is not documented. I have now looked more into it, and I think that the locale should actually be set correctly from the moment factory plugin. I will look more into this.